kuksa-client/kuksa_client/cli_backend/grpc.py

```diff
@@ -144,12 +145,12 @@ def setValues(self, updates: Dict[str, Any], attribute="value", timeout=5):
         return json.dumps({"error": "Invalid Attribute"})
 
     # Function for authorization
-    def authorize(self, token_or_tokenfile:str =None, timeout=5):
+    def authorize(self, token_or_tokenfile:Optional[str] =None, timeout=5):
         if token_or_tokenfile is None:
             token_or_tokenfile = self.token_or_tokenfile
         if os.path.isfile(token_or_tokenfile):
-            token_or_tokenfile = pathlib.Path(token_or_tokenfile)
-            token = token_or_tokenfile.expanduser().read_text(encoding='utf-8').rstrip('\n')
+            token_or_tokenfile_path = pathlib.Path(token_or_tokenfile)
```

Comment:
why do we need this renaming here?

Reply:
It is mypy that does not like when you change type of a variable. `token_or_tokenfile` was first a `str` and then we did an assignment changing it to `Path`. Now instead we use a different variable name.

_error: Incompatible types in assignment (expression has type "Path", variable has type "str")  [assignment]_

Reply:
Ah, now I see :)